### DISCUSSION TITLE
新人想要加入MAA建设该怎么办

### DISCUSSION THREAD

**Discussion by 134820134820** [MaaAssistantArknights/MaaAssistantArknights#3360] [Q&A]
新人想要加入MAA建设，会编程（主要用的Python），C++基本语法看得懂，但是以整个MAA体量，感觉想要搞懂MAA是如何跑起来的还是有点~~苦难~~困难

**Comment by ABA2396** [comment]
[开发者群（QQ 群）](https://jq.qq.com/?_wv=1027&k=JM9oCk3C)
